blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedMultiIngestionHandler.java

```diff
@@ -64,107 +66,115 @@ public void handle(ChannelHandlerContext ctx, FullHttpRequest request) {
 
         requestCount.inc();
 
-        try {
+        final Timer.Context timerContext = handlerTimer.time();
+        long ingestTime = clock.now().getMillis();
 
-            final Timer.Context timerContext = handlerTimer.time();
-            long ingestTime = clock.now().getMillis();
-
-            // this is all JSON.
-            String body = null;
-            try {
-                body = request.content().toString(Constants.DEFAULT_CHARSET);
-                List<AggregatedPayload> bundleList = createBundleList(body);
-
-                if (bundleList.size() > 0) {
-                    // has aggregated metric bundle in body
-                    // convert and add metric bundle to MetricsCollection if valid
-                    MetricsCollection collection = new MetricsCollection();
-                    List<String> errors = new ArrayList<String>();
-
-                    // for each metric bundle
-                    for (AggregatedPayload bundle : bundleList) {
-                        // validate, convert, and add to collection
-                        List<String> bundleValidationErrors = bundle.getValidationErrors();
-                        if (bundleValidationErrors.isEmpty()) {
-                            // no validation error, add to collection
-                            collection.add(PreaggregateConversions.buildMetricsCollection(bundle));
-                        } else {
-                            // failed validation, add to error
-                            errors.addAll(bundleValidationErrors);
-                        }
-
-                        if (bundle.hasDelayedMetrics(ingestTime)) {
-                            Tracker.getInstance().trackDelayedAggregatedMetricsTenant(bundle.getTenantId(),
-                                    bundle.getTimestamp(),
-                                    bundle.getDelayTime(ingestTime),
-                                    bundle.getAllMetricNames());
-                            bundle.markDelayMetricsReceived(ingestTime);
-                        }
+        // this is all JSON.
+        String body = null;
+        try {
+            // block until things get ingested.
```

Comment:
Can we remove this comment? When I was working on my PR, Richard asked what is this code blocking for? I think the comment doesn't make sense and is confusing ppl.